app/scripts/modules/core/src/pipeline/config/triggers/pubsub/pubsub.trigger.ts

```diff
@@ -0,0 +1,44 @@
+import {IController, module} from 'angular';
+
+import {SETTINGS} from 'core/config/settings';
+import {PIPELINE_CONFIG_PROVIDER} from 'core/pipeline/config/pipelineConfigProvider';
+import {SERVICE_ACCOUNT_SERVICE, ServiceAccountService} from 'core/serviceAccount/serviceAccount.service';
+import {IPubsubTrigger} from '@spinnaker/core';
```

Comment:
We don't use `@spinnaker/core` from inside the `core` module. (`@spinnaker/core` is the npm module for the core directory). Use `core/domain/ITrigger` instead.